packages/gatsby-plugin-hoofd/gatsby-ssr.js

```diff
@@ -2,14 +2,18 @@ import { createElement } from 'react';
 import { toStatic } from 'hoofd';
 
 export const onRenderBody = ({ setHeadComponents, setHtmlAttributes }) => {
-  const { title, metas, lang, links } = toStatic();
+  const { title, metas, lang, links, amp, ampScript } = toStatic();
 
   if (lang) {
-    setHtmlAttributes({ lang });
+    setHtmlAttributes({ lang, amp });
   }
 
   setHeadComponents(
     [
+      amp && createElement('script', {
+        async: true,
```

Comment:
Glad to see this, thanks for ensuring it wasn't possible to override the `async` attribute.